lib-py/casmutils/xtal/lattice.py

```diff
@@ -1,15 +1,242 @@
 from . import _xtal
+from . import globaldef
 
-class _Lattice(_xtal.Lattice):
+class Equals:
+
+    """A wrapper class for _xtal.LatticeEquals_f"""
+
+    def __init__(self, ref_lattice, tol):
+        """Construct Equals from Lattice or MutableLattice &
+        a given tolerance
+
+        Parameters
+        ----------
+        ref_site : Lattice or MutableLattice
+        tol : double
+
+        """
+        self._LatticeEquals_f = _xtal.LatticeEquals_f(ref_lattice._pybind_value, tol)
+
+    def __call__(self, other):
+        """Overloading () operator
```

Comment:
`__call__` is always the `()` overload. You should explain what it does or just list only the parameters, and explain what it does in the main documentation for the class.